pkg/agent/apiserver/handlers/ovstracing/handler.go

```diff
@@ -380,7 +380,9 @@ func HandleFunc(aq querier.AgentQuerier) http.HandlerFunc {
 				// ovs-appctl has been executed but returned an error (e.g. the provided
 				// "flow" expression is incorrect). Return the error output to the client in
 				// this case.
```

Comment:
the comment says that we return the error output to the client. It doesn't seem to be the case anymore, even though I think it's what we should do.

Reply:
Changed it to return the error message with `StatusInternalServerError`.